css/css-basics_social-card/css/styles.css

```diff
@@ -1,3 +1,22 @@
 * {
   box-sizing: border-box;
 }
+img {
+  border-radius: 5px;
+}
+button {
+  border: none;
+  background-color: white;
+}
+article {
+  font-family: Impact, Haettenschweiler, "Arial Narrow Bold", sans-serif;
+}
+p {
+  font-family: Arial, Helvetica, sans-serif;
+}
+path {
```

Comment:
I didn't find the css-selector path. I wonder if it worked to change the color of the svg-arrow. Did it work?